examples/xgboost/data_split_gen.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+mkdir data_splits
+for split_mode in uniform exponential
+do
+  python3 utils/prepare_data_split.py --site_num 5 --split_method ${split_mode} --out_path "data_splits/data_split_5_${split_mode}.json"
+done
+
+for split_mode in uniform square
+do
+  python3 utils/prepare_data_split.py --site_num 20 --split_method ${split_mode} --out_path "data_splits/data_split_20_${split_mode}.json"
+done
```

Comment:
Can you combine your `job_config_gen.sh` with `data_split_gen.sh`?

For example. for split_mode of uniform, for 5 sites:
We generate the data split first and then generate the job config using that data_split.

Reply:
I think it is better to separate them, because they are not 1-1: one data split can be used by several job configs. So in some sense the data split is more "general" and potentially can be used by other training schemes.